ui/src/AppTimestamps/AppTimestampsPicker.svelte

```diff
@@ -0,0 +1,193 @@
+<script lang="ts">
+  import {createEventDispatcher} from 'svelte';
+  import TimestampTableRow from './TimestampTableRow.svelte';
+  import {MAX_TIMESTAMPS, MIN_TIMESTAMPS} from './timestamp-helpers';
+  import type {BenchmarkIdentifier, TimestampViewModel} from '../types';
+  import {errorStore} from '../utility/stores';
+
+  export let timestamps: TimestampViewModel[];
+  export let selected: BenchmarkIdentifier[] = [];
+  export let latest: number = undefined;
+
+  type DispatcherEvents = {
+    change: BenchmarkIdentifier[];
+    close: undefined;
+  };
+  const dispatch = createEventDispatcher<DispatcherEvents>();
+
+  let useLatest = !isNaN(latest);
+
+  let recentCount = Math.min(
+    MAX_TIMESTAMPS,
+    Math.max(latest || 0, MIN_TIMESTAMPS)
+  );
+
+  $: maxSelected = selected.length >= MAX_TIMESTAMPS;
+  $: {
```

Comment:
I know I'm being picky, but to this would be so much more concise like this:

```js
$: latest = useLatest ? recentCount : undefined;
```

Reply:
Fixed 81f09d1f2c1c96fec2c21b4adcd91e2251949a1d